src/classes/exec-sync-utils.ts

```diff
@@ -0,0 +1,16 @@
+import { execSync, ExecSyncOptionsWithBufferEncoding } from "child_process";
+
+/**
+ * runs the given command in a shell, vulnerable to injection
+ * 
+ * @param {string} command command to be run in shell
+ * @param {ExecSyncOptionsWithBufferEncoding} options options to be provided to the shell
+ */
+export const runExecSync = async (command: string, options?: ExecSyncOptionsWithBufferEncoding): Promise<string | void> => {
+  try {
+    return execSync(command, options)?.toString(); //NOSONAR
```

Comment:
```suggestion
    return execSync(command, options?).toString(); //NOSONAR
```

Reply:
question mark is outside execSync() to tell toString() that it might not get anyhting to convert
would crash on undefined otherwise

Reply:
```suggestion
    return execSync(command, options?)?.toString(); //NOSONAR
```

Reply:
this does not seem to be the proper syntax, linter disagrees